src/detect-dce-stub-data.c

```diff
@@ -79,8 +79,8 @@ static InspectionBuffer *GetSMBData(DetectEngineThreadCtx *det_ctx,
             return NULL;
         SCLogDebug("have data!");
 
-        InspectionBufferSetup(det_ctx, list_id, buffer, data, data_len);
-        InspectionBufferApplyTransforms(det_ctx, buffer, transforms);
+        InspectionBufferSetupAndApplyTransforms(
+                det_ctx, list_id, buffer, data, data_len, transforms);
```

Comment:
Can you use `DetectHelperGetData` to save even more lines ?

Reply:
Yes -- I think that pattern could be used in several places but is probably best done in a separate PR, imo

Reply:
So, why do you need this commit ? if it "is probably best done in a separate PR"

Reply:
So, why do you need this commit ? if it "is probably best done in a separate PR"